lib/node/runner/checker.go

```diff
@@ -731,3 +737,15 @@ func finishInflation(st *storage.LevelDBBackend, opb operation.Inflation, log lo
 
 	return
 }
+
+func finishUnfreezeRequest(st *storage.LevelDBBackend, tx transaction.Transaction, opb operation.UnfreezeRequest, log logging.Logger) (err error) {
```

Comment:
This does not do anything, does it ?

Reply:
@Geod24 Yes, it check only existence of source account.

Reply:
But that's already done before, isn't it ?

Reply:
@Geod24 Yes that's right. I leaved only log.